src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -42,6 +41,13 @@ public class PSVersionInfo
 
         private static readonly PSVersionHashTable s_psVersionTable;
 
+        /*
+            // The property and fields are generated by PSVersionInfoGenerator source generator.
```

Comment:
Maybe tweak to `The following property ...`

Reply:
Fixed.